sdccc/src/main/java/com/draeger/medical/sdccc/tests/glue/direct/DirectSubscriptionHandlingTest.java

```diff
@@ -545,12 +570,34 @@ private void subscribeToReportWithTheAbilityToFail(final String baseURI,
             soapResponse = requestResponseClient.sendRequestResponse(subscribeRequest);
             if (soapResponse.isFault()) {
                 throw new RuntimeException("Could not subscribe to " + reportName + ".");
+            } else {
+                final SubscribeResponse responseBody = soapUtil
+                    .getBody(soapResponse, SubscribeResponse.class)
+                    .orElseThrow(() -> new MalformedSoapMessageException("Cannot read WS-Eventing Subscribe response"));
+
+                reportTestData.setSubscription(
+                    new SubscribeResult(determineSubscriptionIdForAction(action), responseBody.getExpires()));
```

Comment:
this does not work, because the subscription id is unknown to the event sink

Reply:
Restructured the Test to use Interceptors. This should resolve the problem as Interceptors allow the normal EventSink to be used for all reports.